broker/data-access-layer/eventmesh/src/ApiServerClient.js

```diff
@@ -77,24 +75,49 @@ function convertToHttpErrorAndThrow(err) {
 }
 
 class ApiServerClient {
-  constructor() {	
+  constructor() {
     this.ready = false;
+    this.apiClients = {};
     this.init();
   }
 
   init() {
-    return Promise.try(() => {
-      return Promise.map(_.values(config.apiserver.crds), crdTemplate => {
-        apiserver.addCustomResourceDefinition(yaml.safeLoad(Buffer.from(crdTemplate, 'base64')));
```

Comment:
Since the usage of the `crdtemplate` is removed, we do not have to feed in the `crdtemplate` into the settings.yaml, which means, we would have simpler settings.yml for both BOSH based one as well as the Kubernetes helm chart one. We can take up in a followup PR if you feel it is too much to add it here. Let me know.

Reply:
The crdtemplate is also currently used to get the version and plural of osb crds. But these can very easily made constants and crds removed for settings yaml

Reply:
Not just osb crds, for all crds, the version and plural is fetched from the crdtemplate. We use `getCrdJson` and `getCrdVersion(crdJosn)`

Reply:
Yes, let us refactor it in a separate PR